specs/ExtendedAcceleratorKeyPressed.md

```diff
@@ -0,0 +1,187 @@
+                                {
+                                    // Allow the browser to process F7 key
+                                    CHECK_FAILURE(args2->put_AllowBrowserAcceleratorHandling(TRUE));
+                                }
+                            }
+                        }
+                        return S_OK;
+                    })
+                    .Get(),
+                &m_acceleratorKeyPressedToken));
+
+        }
+    }
+```
+
+## C#
+```c#
+    CoreWebView2Settings _webViewSettings = webView.CoreWebView2.Settings;
+    // All browser accelerator keys are disabled with this setting. The default value for the 
+    // `AllowBrowserAcceleratorHandling`property is set to the current value of this setting. 
+    // Changing this setting to `false` will change the default setting for 
+    // AllowBrowserAcceleratorHandling to `false`
+    _webViewSettings.AreBrowserAcceleratorKeysEnabled = false;
+
+    webView.CoreWebView2.AcceleratorKeyPressed += WebView2Controller_AcceleratorKeyPressed;
+
+    void WebView2Controller_AcceleratorKeyPressed(object sender, CoreWebView2AcceleratorKeyPressedEventArgs e)
+    {
+        switch (e.KeyEventKind)
+        {
+            case CoreWebView2KeyEventKind.KeyDown:
+            case CoreWebView2KeyEventKind.SystemKeyDown:
+                {
+                    // Allow the browser to process F7 key
+                    if (e.VirtualKey == Key.F7) 
+                    {
+                        e.AllowBrowserAcceleratorHandling = true;
+                    }
+                    break;
+                }
+            
+        }
+    }
+
+```
+
+# Remarks
+By default, `CoreWebView2Settings.AreBrowserAcceleratorKeysEnabled` is `TRUE` and default value for
+`AllowBrowserAcceleratorHandling` is `TRUE`. When developers set  
+`CoreWebView2Settings.AreBrowserAcceleratorKeysEnabled` to `FALSE` this will also change the default 
+value for `AllowBrowserAcceleratorHandling` to `FALSE`. If developers want specific keys to be handled 
+by the browser after changing the `CoreWebView2Settings.AreBrowserAcceleratorKeysEnabled` settings to 
+`FALSE`, they need to enable them by setting `event_args->put_ AllowBrowserAcceleratorHandling(TRUE)`. 
+This option will give the event arg higher priority over the settings when the browser decide whether 
+to handle the keys.  
+
+# API Details
+## C++
+```
+/// This is a continuation of the ICoreWebView2AcceleratorKeyPressedEventArgs interface.
+[uuid(45238725-3774-4cfe-931f-7985a1b5866f), object, pointer_default(unique)]
+interface ICoreWebView2AcceleratorKeyPressedEventArgs2 : ICoreWebView2AcceleratorKeyPressedEventArgs {
+  /// This property allows developers to enable or disable the browser from handling a specific 
+  /// browser accelerator key such as Ctrl+P or F3, etc.
+  ///
+  /// Browser accelerator keys are the keys/key combinations that access features specific to 
+  /// a web browser, including but not limited to:

```

Comment:
All of this info in this long comment block here is great. I didn't really understand what this feature was about after reading the Background and Description sections. It was only when I read this comment block that I understood.

Would it be better to move some of this info out of the api comment and into the main section of the doc? 

I don't have strong opinions either way, but we should make sure that this info shows up clearly on the docs.microsoft.com page for this api.

Reply:
The interaction between the AllowBrowserAcceleratorHandling and Handled properties is very important to understand and should be called out in real docs, not hidden in a comment.

Reply:
updated!